### PR TITLE
Delay time in banner disappearing resolved

### PR DESCRIPTION
**Pull request by emmacode**
Resolved the delay issue when cookie banner is clicked.
When you click on "ok", the banner disappears immediately

Issue ==> https://github.com/gbenga504/music-port/issues/32

### CODE REVIEW

File: packages/frontend-app/src/app/components/CookieBanner.tsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
 export const CookieBanner = () => {
   const [value, consentToCookie] = useCookieConsent();
+  const [isVisible, setIsVisible] = useState(false);
 
-  return !value ? (
+  useEffect(() => {
+    if (!value) {
+      setIsVisible(false);
+    }
+  }, [value]);
```
Comment (gbenga504) [inline packages/frontend-app/src/app/components/CookieBanner.tsx:15]: This is not necessary. Having this causes a second render after mount which is not necessary.

File: packages/frontend-app/src/app/components/CookieBanner.tsx [hunk 2]
```diff
@@ -1,13 +1,25 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import classNames from "classnames";
 
 import { Button } from "./Button";
 import useCookieConsent from "../hooks/useCookieConsent";
 
 export const CookieBanner = () => {
   const [value, consentToCookie] = useCookieConsent();
+  const [isVisible, setIsVisible] = useState(false);
```
Comment (gbenga504) [inline packages/frontend-app/src/app/components/CookieBanner.tsx:9]:
```suggestion
  const [isVisible, setIsVisible] = useState(!!value);
```

Doing this is better than having a useEffect just to set the visible state. Also the component does not go through any unnecessary re-render

**Review by gbenga504 — CHANGES_REQUESTED** [review]
Nice work 🔥 , just a few suggestions.

File: packages/frontend-app/src/app/components/CookieBanner.tsx (hunk 1 above)
Comment (emmacode) [reply packages/frontend-app/src/app/components/CookieBanner.tsx:15]: ok, thanks

File: packages/frontend-app/src/app/components/CookieBanner.tsx (hunk 2 above)
Comment (emmacode) [reply packages/frontend-app/src/app/components/CookieBanner.tsx]: Changes have been applied accordingly. The approach reduces the number of renders and makes the code cleaner by handling the initial visibility directly within the useState initialization.

**Comment by emmacode** [comment]
Changes have been applied accordingly. The approach reduces the number of renders and makes the code cleaner by handling the initial visibility directly within the useState initialization.

**Review by gbenga504 — APPROVED** [review]
LGTM! Thanks alot for the contribution

**Pull request merged**